docs/atlascli/command/atlas-deployments-connect.txt

```diff
@@ -55,10 +55,30 @@ Options
      - string
      - false
      - Method for connecting to the deployment. Valid values are mongosh, compass, and skip.
+   * - --connectionStringType
+     - string
+     - false
+     - When set to 'private', retrieves the connection string for the network peering endpoint. This value defaults to "standard".
```

Comment:
I don't see where this line was defined in the code, so leaving this comment here. We're using both single quotes and double quotes here. I think we should settle on double quotes.

also a question: are `private' and `standard` the only accepted values?
```suggestion
     - When set to "private", retrieves the connection string for the network peering endpoint. This value defaults to "standard".
```

Reply:
the first half of the message comes from https://github.com/mongodb/mongodb-atlas-cli/blob/a100b707ff7f008bc12fe18cf54a2a31206fa317/internal/usage/usage.go#L51 and is shared with [atlas-clusters-connectionStrings-describe](https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-clusters-connectionStrings-describe/). 
Second half comes from cobra (the CLI lib).

I can make the change and use double quotes ("private") but looking at usage.go there are more messages using single quotes (e.g. 'forever'). Do you think we should update those too?

For the second question, yes, private and standard are the only options for this flag

Reply:
Merging this PR and will create a separate PR based on the suggestions that will result from this thread